### PR TITLE
Lg 1473 Pagination of Organizations

### PR DESCRIPTION
**Pull request by lgarceau768**
## Change Details
Adding pagination to the `client_tokens` and `public_keys` to display a max of 10 records per page.
Using url parameters on the `localhost:3900/?` request, in order to accomplish this (`token_page and key_page`)
Added a new ruby gem [kaminari](https://github.com/kaminari/kaminari)
Used the template generator from kaminari to apply the pagination functionality to the [uswds pagination component](https://designsystem.digital.gov/components/pagination/)
* 

## Security Implications
Does not connect to the database calls at all. Only manipulating the array of client tokens and public keys after it has already been retrieved (purely ruby on rails logic)
- [x] new software dependencies
No major issues within the dependency [(found here)](https://codeclimate.com/github/kaminari/kaminari/issues?severity%5B%5D=major&status%5B%5D=&status%5B%5D=open&status%5B%5D=confirmed)

## Todo
https://jira.cms.gov/browse/DPC-2818

# ScreenShots
Screen Shot 2022-08-30 at 12 12 22 PM
Screen Shot 2022-08-30 at 12 12 11 PM
Screen Shot 2022-08-30 at 12 37 01 PM
Screen Shot 2022-08-30 at 12 37 51 PM

# Testing Script
```
#!/bin/sh
mkdir ~/cred_local
cd ~/cred_local
openssl genrsa -out private.pem 4096
openssl rsa -in private.pem -outform PEM -pubout -out public.pem
openssl dgst -sign private.pem -sha256 -out snippet.txt.sig snippet.txt
openssl dgst -verify public.pem -sha256 -signature snippet.txt.sig snippet.txt
openssl base64 -in snippet.txt.sig -out signature.sig
echo "key name"
echo $RANDOM | base64 | head -c 20; echo
echo "signature"; echo
cat signature.sig
echo "pk"; echo
cat public.pem
cd ../
```

### CODE REVIEW

File: dpc-web/app/views/portal/_assigned_content.html.erb [hunk 1]
```diff
@@ -185,12 +186,13 @@
                     </td>
                   </tr>
                 <% end %>
-              </tbody>
+              </tbody>
```
Comment (mftcodes) [inline dpc-web/app/views/portal/_assigned_content.html.erb:189]: Remove Whitespace

File: dpc-web/app/views/portal/_assigned_content.html.erb [hunk 2]
```diff
@@ -99,12 +99,13 @@
...
```
Comment (mftcodes) [inline dpc-web/app/views/portal/_assigned_content.html.erb:102]: Remove whitespace

**Review by mftcodes — CHANGES_REQUESTED** [review]
Other than the white space, this looks good to me.

**Review by TiannaW24 — CHANGES_REQUESTED** [review]
Functionality looks good. Can we add one thing: we need an indicator showing which page the user is currently on.

File: dpc-web/app/assets/stylesheets/utilities/_add-ons.scss [hunk 3]
```diff
@@ -8,4 +8,9 @@
 
 .ds-u-color--gray-light {
   color: $color-gray-light;
-}
\ No newline at end of file
+}
+
+.dpc-current-page > *{
+  background-color: black;
+  color: white !important;
```
Comment (mftcodes) [inline dpc-web/app/assets/stylesheets/utilities/_add-ons.scss:15]: Anyway to increase specificity here rather than using `!important`?

**Review by TiannaW24 — APPROVED** [review]

File: dpc-web/app/assets/stylesheets/utilities/_add-ons.scss (hunk 3 above)
Comment (TiannaW24) [reply dpc-web/app/assets/stylesheets/utilities/_add-ons.scss:15]: +1 to this

**Review by mftcodes — APPROVED** [review]

**Pull request merged**